app/views/system/nodes/index.scala.html

```diff
@@ -159,6 +159,7 @@
                     <div class="bar heap-total-percent" style="width: @(jvm.totalMemoryPercentage()-jvm.usedMemoryPercentage)%"></div>
                 </div>
 
+                <strong><span class="journal-uncommitted">@nodes.get(jvm.getNodeId).getJournalInfo().uncommittedJournalEntries</span> unprocessed messages</strong> in the journal.
```

Comment:
Also confusing having an element called `node-heap-usage` with journal information inside. Please change the container name if it is not too hard. I know how those small changes sometimes end up :)